scripts/mlp/utils/trajectories.py

```diff
@@ -609,11 +609,11 @@ def __init__(self,curves,placement_init,placement_end,time_interval):
     self.t_total = curves.max()
     assert abs(self.t_total - (time_interval[1] - time_interval[0])) <= 1e-4, "time interval is not coherent with the length of the Bezier curves"
     assert len(curves.times)%2 == 0, "PolyBezier object contain an even number of curves, not implemented yet."
-    id_mid = len(curves.times)/2
+    id_mid = int(len(curves.times)/2)
     # retrieve the timings of the middle segment (duration and begin/end wrt to the other curves)
     self.t_mid_begin = curves.times[id_mid-1]
     self.t_mid_end = curves.times[id_mid]
-    self.t_mid = curves.curves[curves.numCurves()/2].max()
+    self.t_mid = curves.curves[int(curves.numCurves()/2)].max()
```

Comment:
You can also use the `//` operator for *divisions entières* (don't know how to say that…)

Reply:
"integer division".

Reply:
Ok, changes done.